.circleci/config.yml

```diff
@@ -694,30 +699,31 @@ jobs:
           command: |
             cp "playwright-e2e/config/.env.<< parameters.env >>" .env
             cat .env | awk '/^[^#]/ {print "export " $0}' >> "$BASH_ENV"
-
             # export env from file (created in build-playwright-job)
             cat playwright-env/envvars | awk '{print "export " $0}' >> "$BASH_ENV"
-
             source "$BASH_ENV"
+      - node/install:
+          install-yarn: false
+          node-version: '18.13.0' # LTS version
```

Comment:
To fix a new error found only on CircleCI.
```
adult-enrollment.spec.ts:24:3 Can complete application @functional @enrollment @rgp
[chromium] › tests/rgp/adult-enrollment.spec.ts:24:3 › Adult Self Enrollment › Can complete application @functional @enrollment @rgp 

    ReferenceError: fetch is not defined

       at utils/api-utils.ts:65

      63 |   const credentials = JSON.parse(buildAuth0ClientCredentials(app));
      64 |
    > 65 |   return fetch(`https://${credentials.domain}/oauth/token`, {
         |   ^
      66 |     method: 'POST',
      67 |     headers: { 'content-Type': 'application/x-www-form-urlencoded' },
      68 |     body: new URLSearchParams({

        at getAuth0AccessToken (/home/circleci/repo/playwright-e2e/utils/api-utils.ts:65:3)
        at setAuth0UserEmailVerified (/home/circleci/repo/playwright-e2e/utils/api-utils.ts:121:49)
        at /home/circleci/repo/playwright-e2e/tests/rgp/adult-enrollment.spec.ts:59:5
```

Reply:
Oh, cool!